modules/TypeDefinitions.js

```diff
@@ -20,7 +20,11 @@ export type RouteProps = {
   children?: (props: ContextRouter) => React$Element<any>,
   path: string,
   exact?: boolean,
-  strict?: boolean
+  strict?: boolean,
+  state?: NavigationState<{
```

Comment:
But this is wrong because `state` must just be an `Object` not a `NavigationState`